scripts/run-integration-tests.sh

```diff
@@ -2,14 +2,13 @@
 
 PROJECT_ROOT="$(dirname "$(dirname "$(realpath "$0")")")"
 cd "${PROJECT_ROOT}" || exit 1
+./scripts/cleanup-test-processes.sh
```

Comment:
I was hesitant to do this originally, as there is some minor risk of killing a process that you wanted, but if you agree that the benefit exceeds the risk, I'll leave this addition here and in the unit test script.

Reply:
seems okay to me. does that script still need to manually kill `monero-wallet-rpc` now that it's controlled by swapd?

Reply:
For the integration tests, it doesn't need to, as the integration tests are using randomized JSON-RPC port for the `monero-wallet-rpc` instances and the wallets are created in a temporary directory.  On Linux, I haven't seen `monero-wallet-rpc` instances killed (since killing `swapd` sends them SIGTERM), but if you did a forced shutdown of `swapd` on MacOS, the wallet instances probably won't stop. If you see the script killing `monero-wallet-rpc` instances, let me as much information about how it happened as you can so I can look into it. In answer to your question, at least on Linux, we can probably remove that kill.